app/templates/photo-albums/photo-album/photos/show.hbs

```diff
@@ -31,17 +31,17 @@
     {{/link-to}}
 
     {{#if model.hasExif}}
-      <button class="ml-1 btn btn-sm btn-outline-info" data-toggle="collapse" data-target="#collapsibleExif" type="button">
-        {{t 'Show Exif'}}
+      <button class="ml-1 btn btn-sm btn-outline-info" {{action 'toggleShowExif'}}>
```

Comment:
~~Ik denk dat we hier niet onze eigen methode voor hoeven schrijven, maar met en helper~~

Okay er is dus dit: https://github.com/DockYard/ember-composable-helpers/blob/master/README.md#toggle

Leuk, want de ember-composable-helpers gebruik ik ook al in een eigen branch voor het privacy scherm. Ze maken net wat minder eigen code mogelijk 👌

Je zou die #toggle kunnen gebruiken

Reply:
Ah, check, fix ik dat even. Ik gebruikte de oplossing uit de ember documentatie.